lua/wikis/commons/Widget/Tournament/Title.lua

```diff
@@ -0,0 +1,64 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Widget/Tournament/Title
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Class = require('Module:Class')
+local Game = require('Module:Game')
+local LeagueIcon = require('Module:LeagueIcon')
+local Lua = require('Module:Lua')
+
+local Widget = Lua.import('Module:Widget')
+local HtmlWidgets = Lua.import('Module:Widget/Html/All')
+
+local Link = Lua.import('Module:Widget/Basic/Link')
+
+---@class TournamentsTickerLabelWidget: Widget
+---@operator call(table): TournamentsTickerLabelWidget
+
+local TournamentsTickerTitleWidget = Class.new(Widget)
```

Comment:
```suggestion
---@operator call(table): TournamentsTickerLabelWidget
local TournamentsTickerTitleWidget = Class.new(Widget)
```

Reply:
For some reason that then leads to annotation error "duplicate field" on the definition of :render, any ideas on why?